### PR TITLE
Bug 1210303 - marionette-extension should support v2.5

### PR DESCRIPTION
**Pull request by ypwalter**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -35,3 +36,28 @@ and use its install function to install marionette via a script. Example::
   from marionette_extension import install  
   install(FXOS_VERSION, adb=ADBPATH) # Replace FXOS_VERSION with a supported version, and you may use ADBPATH to specify your adb path. If not specified, it will assume 'adb' is on the $PATH
 
+
+How to add a new version?
+==============
```
Comment (askeing) [inline README.rst]: Should have the same length of title.

Comment (ypwalter) [inline README.rst]: Thank you.

**Comment by ypwalter** [comment]
@askeing, check-in needed.

**Comment by askeing** [comment]
merged due to r+ at Bug 1210303

**Pull request merged**